java-rx-smgrsdk/src/main/java/io/vantiq/svcconnector/StorageManagerVerticle.java

```diff
@@ -81,116 +81,119 @@ public Completable rxStop() {
-                            .toFlowable();
-                    break;
-                case "select":
-                    result = storageManager.select((String) msg.params.get("storageName"),
-                            (Map<String, Object>) msg.params.get("storageManagerReference"),
-                            (Map<String, Object>) msg.params.get("properties"), (Map<String, Object>) msg.params.get("qual"),
-                            (Map<String, Object>) msg.params.get("options"));
-                    break;
-                case "aggregate":
-                    result = storageManager.aggregate((String) msg.params.get("storageName"),
-                            (Map<String, Object>) msg.params.get("storageManagerReference"),
-                            (List<Map<String, Object>>) msg.params.get("pipeline"),
-                            (Map<String, Object>) msg.params.get("options"));
-                    break;
-                case "selectOne":
-                    result = storageManager.selectOne((String) msg.params.get("storageName"),
-                                    (Map<String, Object>) msg.params.get("storageManagerReference"),
-                                    (Map<String, Object>) msg.params.get("properties"), (Map<String, Object>) msg.params.get("qual"),
-                                    (Map<String, Object>) msg.params.get("options"))
-                            .toFlowable();
-                    break;
-                case "delete":
-                    result = storageManager.delete((String) msg.params.get("storageName"),
-                                    (Map<String, Object>) msg.params.get("storageManagerReference"),
-                                    (Map<String, Object>) msg.params.get("qual"), (Map<String, Object>) msg.params.get("options"))
-                            .toFlowable();
-                    break;
-                case "getTypeRestrictions":
-                    result = storageManager.getTypeRestrictions().toFlowable();
-                    break;
-                case "initializeTypeDefinition":
-                    if (msg.params == null || !(msg.params.get("proposedType") instanceof Map)) {
-                        result = Flowable.error(new Exception("unrecognized storage manager service procedure call: " +
-                                msg.procName));
-                    } else {
-                        //noinspection unchecked
-                        result = storageManager.initializeTypeDefinition(
-                                (Map<String, Object>) msg.params.get("proposedType"),
-                                (Map<String, Object>) msg.params.get("existingType")
+        try {
+            if (msg.procName != null) {
+                // procedure name is <package name>.<service name>.<procedure name>
+                String[] parts = msg.procName.split("\\.");
```

Comment:
seems like Git got quite confused by the indentation...

Reply:
sorry, the diff looked more reasonable in the IDE